docs/using-kontena/nodes.md

```diff
@@ -42,6 +42,36 @@ The re-scheduling of grid services will happen after the node offline grace peri
 - `ha`: 2 minutes
 - `random`: 30 seconds
 
+### Node availability
+
+Node availablity for scheduling services can be controlled with `kontena node availablity` command. The current supported states are: `drain`and `active`.
+
+Nodes availablity status can be seen using `kontena node show xyz`:
+```
+$ k node show moby
+moby:
+  id: BCKY:WMSM:IMW4:KSNQ:C2MZ:4EZ7:ZG7J:5UQ2:MMA3:NYK7:5PEF:JBLN
+  agent version: 1.3.0.dev
+  docker version: 1.13.1
+  connected: yes
+  availability: drain
+  last connect: 2017-05-15T06:03:01.720Z
+  last seen: 2017-05-15T06:03:00.523Z
+... snip ...
+```
+
+#### Active
+
+Kontena scheduler can assign tasks into any node marked as `active` availability. `active` is also the default state for all nodes. To re-activate a node use `kontena node availability xyz active`
+
+#### Drain
+
+For a planned maintenance or node decommisioning it is a good idea to first drain the node so there are minimal disruptions on the services running. To drain the services from a node use `kontena node availability xyz drain`
```

Comment:
CLI would have to figure out which service instances should be left on the node. Plus the corner case of draining the last node in the grid. Maybe enhancement for another PR?